src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -105,7 +104,7 @@ public void testGetSnapshot() throws InvalidProtocolBufferException {
       List.of(TripUpdate.parseFrom(cancellation)),
       feedId
     );
-    assertSame(snapshot, updater.getTimetableSnapshot());
+    assertNotSame(snapshot, updater.getTimetableSnapshot());
```

Comment:
I think it's actually that you are setting the refresh rate to Duration.ZERO that is causing this change?

It's not very clear what this was testing in the first place. But my guess was that it was testing that you get the same snapshot if you query it within a short period of time even if there was a modification.

With the new assertion it's maybe even less clear what behaviour we are testing. Are we testing that you get a new snapshot if you apply updates? In that case we are already sort of testing that in the first part of this test.

IMO we can remove this part of the test. And otherwise we should add a comment about what it is verifying.

Reply:
I also find it quite confusing and would be very happy to remove the test.

Reply:
251e73b3e51c99c6d5a051844140b5e71ef57081